setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
I think it could be due to mixing of easy_install and pip for package installations, similar to [here](http://stackoverflow.com/questions/6200056/pip-broke-how-to-fix-distributionnotfound-error). Don't know if this is still relevant as pip nowadays comes with Python afaik.